mlrun/runtimes/nuclio/application/application.py

```diff
@@ -665,3 +696,12 @@ def _sync_api_gateway(self):
         self.status.api_gateway = APIGateway.from_scheme(api_gateway_scheme)
         self.status.api_gateway.wait_for_readiness()
         self.url = self.status.api_gateway.invoke_url
+
+    def _resolve_default_api_gateway_name(self):
+        api_gateway_name = (
+            f"{self.metadata.name}-{self.metadata.tag}"
+            if self.metadata.tag
+            else self.metadata.name
+        )
+        api_gateway_name += "-default"
```

Comment:
are we sure we want this `-default` suffix on every default app runtime gw?
I feel the default should reflect the common use case, and the common use case would be just the name of the function with the tag.
Since we have the validation in the `create_api_gateway` method, there shouldn't be overlap issues, so I don't think this is needed.